test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -531,6 +531,18 @@ def expected_bigquery_complex_catalog(self):
                 'stats': clustering_stats,
                 'columns': self._clustered_bigquery_columns('DATE'),
             },
+            'model.test.multi_clustered': {
+                'unique_id': 'model.test.multi_clustered',
+                'metadata': {
+                    'comment': None,
+                    'name': 'multi_clustered',
+                    'owner': None,
+                    'schema': my_schema_name,
+                    'type': 'table'
+                },
+                'stats': clustering_stats,
```

Comment:
This PR is failing because `clustering_stats` is defined as:
```
 clustering_stats = self._bigquery_stats(True, partition='DAY', cluster='first_name')
```

but for this multi-clustered, model, we want:
```
multi_clustering_stats = self._bigquery_stats(True, partition='DAY', cluster='first_name,email')
```

Can you change this to:
```suggestion
                'stats': multi_clustering_stats,
```

and create a new `multi_clustering_stats` variable next to `clustering_stats` above?